engine/class_modules/sc_death_knight.cpp

```diff
@@ -9534,7 +9542,7 @@ struct death_knight_module_t : public module_t {
 
   void register_hotfixes() const override
   {
-    hotfix::register_spell( "Death Knight", "2019-03-12", "Incorrect cooldown for Magus of the Dead's Frostbolt.", 288548 )
```

Comment:
Updated the existing hotfix instead of making a new one since I assume we have no need to leave the azerite trait hotfix enabled for Shadowlands